readme.md

```diff
@@ -26,6 +26,26 @@ const terminalImage = require('terminal-image');
 })();
 ```
 
+Optionally, you can specify the `height` and/or `width` to scale the image.
```

Comment:
The new options needs to be properly documented in the `API` section. The readme and TypeScript file should be kept in sync.